pkg/compute/serve.go

```diff
@@ -28,6 +28,7 @@ type ServeCommand struct {
 	build            *BuildCommand
 	env              cmd.OptionalString
 	file             string
+	addr             string
```

Comment:
Similarly I've been sorting keys (separate from any embedded structs, e.g. `cmd.Base`) as it makes it easier to find things.